caffe2/ideep/operators/conv_op.cc

```diff
@@ -36,95 +36,98 @@ class IDEEPConvOp : public IDEEPConvPoolOpBase {
     const auto& X = Input(INPUT_X);
     const auto& filter = Input(FILTER);
     auto* Y = Output(OUTPUT);
-    auto grouped = filter.is_grouped() ? 1 : 0;
-    auto Y_dims_conv = CalcOutputDims(
-        X,
-        grouped ? (filter.get_dim(0) * filter.get_dim(1)) : filter.get_dim(0));
 
     CAFFE_ENFORCE(4 == X.ndims());
-    CAFFE_ENFORCE(4 == filter.ndims() || (grouped && (group_ > 1)));
```

Comment:
same concern here https://github.com/pytorch/pytorch/pull/32422#discussion_r371382516

Reply:
We haven't serialized caffe2 models before, so I don't think it could be a problem as the 5-d weights only exist at runtime.